### DISCUSSION TITLE
LAMMPS with MACE: create_lammps_model.py doesn't exist

### DISCUSSION THREAD

**Discussion by ShubhangG** [ACEsuit/mace#213] [Q&A] [answered]
Hello, I have been following the [MACE in LAMMPS documentation](https://mace-docs.readthedocs.io/en/latest/guide/lammps.html). I think the `create_lammps_model.py` script is no longer available in this MACE repository. I notice a `lammps_mace.py` class but no script that will convert the mace model to lammps. Was it removed?

**Comment by ilyes319** [comment, marked as answer]
It was moved to mace/cli/create_lammps_model.py for better integration. It should appear as such in the new documentation.

**Reply by ShubhangG** [reply]
I'm sorry I was trying to follow the new documentation but there doesn't seem to be a subfolder `cli` in this repository, is there another one I should take a look at?

Sorry if its an obvious question

**Reply by ilyes319** [reply]
Please look at the develop branch

**Reply by ShubhangG** [reply]
Got it Thank you for your patience 🙏!
